src/routes/docs/advanced/security/ccpa/+page.markdoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title: CCPA
+description: Protecting your and your users' data privacy is a priority at Appwrite. Learn about Appwrite's compliance with the California Consumer Privacy Act (CCPA).
+---
+
+Appwrite is compliant with the California Consumer Privacy Act (CCPA). The CCPA is a privacy law that gives California residents more control over their personal information, helping ensure their data privacy rights.
+
+To confirm Appwrite's compliance with the CCPA, we have ensured the following rights for users:
+
+- **Right to know:** Appwrite users can request information about the personal data that is collected, shared, or sold.
+- **Right to delete:** Users can request that Appwrite delete their personal data, with certain exceptions.
```

Comment:
```suggestion
- **Right to delete:** Users can request that Appwrite delete their personal data, with certain exceptions when it comes to security.
```
This might be better, or something along those lines. Lets make it clear what the exception is - I am guessing it's stuff like IP address when website is being attacked.